### DISCUSSION TITLE
Safety question

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1985] [General]
Does GO rely on any part of the perl or python packages?

**Comment by larspalo** [comment]
No, not in any actual GO build code. There are however a few git/Github pre-commit hooks (clang-format and update-copyright) that use python though, but they are only run when committing code to git (and they can actually, at least locally, be bypassed if one really wants to).

**Comment by jason-barba** [comment]
Note to self: Do not remove python from ubuntu system!

**Reply by larspalo** [reply]
Yes, you definitely don't want to remove python from the ubuntu system...

I believe that you more or less need to build something like LFS (Linux From Scratch) or basically stay away from any general purpose "full functionality" desktop distribution if you really want to avoid python. Perhaps a very basic Debian installation could work, but I've never tested it personally, only used full desktop installations.

**Reply by jason-barba** [reply]
Fortunatly, my son's setup of backup procedure and help with restoring helped. The organ  is up and running :-)
